parser/fuzz/Cargo.toml

```diff
@@ -26,3 +31,9 @@
   path = "fuzz_targets/statements.rs"
   test = false
   doc  = false
+
+[[bin]]
+  name = "structured"
+  path = "fuzz_targets/structured.rs"
```

Comment:
maybe this can be `statements-structured.rs`? 

i'm not sure how to name this, but I named `structured.rs` based off of `../tests/structured.rs`

Reply:
Your choice :slightly_smiling_face: